openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml

```diff
@@ -1,8 +1,1511 @@
 <?xml version='1.0' encoding='utf-8'?>
 <openerp>
-    <data>
-        <record model="res.groups" id="group_user">
-            <field name="implied_ids" eval="[(4, ref('group_no_one'))]"/>
-        </record>
-    </data>
```

Comment:
Travis is red because a test fails. It seems it didn't liked the removal of this lines.

Reply:
So if you restore this it throws error? But before it was green...

Reply:
Here is the explanation: https://github.com/OCA/OpenUpgrade/pull/607

Reply:
OK, my question was why do you remove it, but I see that you are only changing indentation and they appear later.